src/rrtest.py

```diff
@@ -266,26 +266,59 @@ def main():
     config.read(test_dir + "config.ini")
     testname = config.get("rr_recording", "rr_dir")
 
+    # create command set proper environmental variable
+    os.environ['RR_LOG'] = 'ReplaySession'
+    rr_config_replay = ['rr', 'replay', '-a', testname]
+
+    # execute replay command, this time to compare against trace. We write the
+    # ReplaySession output to replaysession.log
+    out_fd = open(test_dir + "replaysession.log", 'wb')
+    proc = subprocess.Popen(rr_config_replay, stdout=out_fd, stderr=out_fd)
+    while proc.poll() is None:
```

Comment:
Hm, is it possible to save the exit code here? It might be useful if we decide to handle the child process failing.

Reply:
`Popen.poll()` sets the `Popen.returncode` attribute (I think) as a side effect. Should I use something like `communicate()` instead of `poll()`?